src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
```

Comment:
It's better to name it in lowercase, basically we use upper + snake keys for static variables that won't be changed